package.json

```diff
@@ -70,6 +70,13 @@
                 }
             ]
         },
+        "viewsWelcome": [
+            {
+              "view": "testExplorer",
+              "contents": "Test explorer will not show test cases in [LightWeight Mode](https://aka.ms/vscjava-lightweight).\n[Swith to Standard Mode](command:java.server.mode.switch)",
```

Comment:
No test cases are listed because the Java Language Server is currently running in Lightweight Mode. To show test cases, click on the button to switch to Standard Mode.